src/files/FilesPage.js

```diff
@@ -70,6 +77,43 @@ class FilesPage extends React.Component {
     this.onRenameCancel()
   }
 
+  onDelete = (files) => {
+    let filesNumber = 0
+    let foldersNumber = 0
+
+    files.forEach(file => {
```

Comment:
Don't you prefer one liners when it's a simple conditional?

```js
files.forEach(file => file.type === 'directory' ? foldersNumber++ : filesNumber++)
```

Reply:
For me it's indifferent, I don't think one way is better than the other. I can change it though.